README.md

```diff
@@ -1,4 +1,15 @@
-# intern2021
+# Journey
+Journey is a learning experience running on [.NET Interactive Notebooks](https://github.com/dotnet/interactive) where teachers can design a step-by-step path for students. Along the student's journey, teachers can create challenges, provide customized feedback, and define rules to continue.
 
+## About Journey
+We believe that the journey towards a goal is one of the most crucial aspects of learning. As students, we wanted to create an engaging notebook experience for the classroom that utilized this idea. 
```

Comment:
"We believe that the journey towards a goal is one of the most crucial aspects of learning." Is this our mission/value statement? That learning is about moving towards a goal?
Throwing something out there:
We model learning as a journey: personalized, interactive, and engaging.

Reply:
"engaging notebook experience": Is "engaging learning experience" more accurate?

Reply:
not sure if the "as students" statement adds value.